voidcrew/_maps/RandomRuins/IceRuins/icemoon_underground_oldstation.dmm

```diff
@@ -4707,25 +4682,29 @@
 	},
 /turf/open/floor/plasteel/white,
 /area/ruin/space/has_grav/ancientstation/proto)
+"kl" = (
+/obj/structure/table/reinforced,
+/obj/machinery/the_singularitygen,
```

Comment:
This should absolutely be a tesla and not a singulo. Space is good for singulo, singulo is bad for planet. If I remember correctly, but I could be wrong, a singulo either tears a hole in a planet, or it doesn't, but it rapidly gains mass from creating items from the fake "plating" of planetary turfs shedding rods into the singulo. All in all, singulo is bad for planets.